product_pricelist_supplierinfo/tests/test_product_supplierinfo.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2018 Tecnativa - Vicent Cubells
+# Copyright 2018 Tecnativa - Pedro M. Baeza
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo.tests import common
+
+
+class TestProductSupplierinfo(common.SavepointCase):
+    at_install = False
```

Comment:
Please remove `at_install` and `post_install`

Reply:
I can't remove it because I have the problem when creating the products that if `product`is installed, it fails by the SQL constraint.

Reply:
@pedrobaeza Tests are green without the post_install. That must be removed. https://travis-ci.org/acsone/product-attribute/jobs/449732838

Reply:
OK, I will apply odoo/odoo#27883 in the systems where I put this module for not fighting you each time.